### ISSUE TITLE
Reduce headers in DNS-over-HTTP(S) requests

### ISSUE THREAD

**Issue by csagan5** [bromite/bromite#70]
For reference see https://mailarchive.ietf.org/arch/msg/doh/vHjITrOMhWSdrozGFe4-eGNMEJc

I can currently identify in Chromium/Bromite that at least the User-Agent and Accept-Language should be replaced when contacting the DoH endpoint.

Other issues (like cookies) are already covered in the Chromium implementation so they do not need a patch, see https://github.com/chromium/chromium/blob/0f26a5815e108f870a08eadc094e86af9830d72b/net/dns/dns_transaction.cc#L436

```c++
    request_->SetLoadFlags(request_->load_flags() | LOAD_DISABLE_CACHE |
                           LOAD_BYPASS_PROXY | LOAD_DO_NOT_SEND_COOKIES |
                           LOAD_DO_NOT_SAVE_COOKIES |
                           LOAD_DO_NOT_SEND_AUTH_DATA);
```

**Comment by csagan5** [comment]
I have implemented a new flag called `LOAD_MINIMAL_HEADERS` that makes sure only `Accept` and `Accept-Encoding` are sent (without any Brotli mention); `User-Agent`, `Referrer` and `Accept-Language` are never sent when this flag is selected.

Reference for the draft changes discussed on the mailing list: https://github.com/nusenu/draft-ietf-doh-dns-over-https/blob/privacy_considerations/draft-ietf-doh-dns-over-https-latest.mkd

**Comment by csagan5** [comment]
Fixed in 67.0.3396.95
